packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -88,8 +88,8 @@ export default function Sidebar({
   });
   const hasSquads = squads?.length > 0;
   const squadsUrl = hasSquads
-    ? squadCategoriesPaths['My Squads']
-    : squadCategoriesPaths.discover;
+    ? `${webappUrl}squads/discover/my`
+    : `${webappUrl}squads/discover`;
```

Comment:
It's probably safer to use the defined constant in case we ever change them.

```suggestion
    ? `${webappUrl}${squadCategoriesPaths['My Squads']}`
    : `${webappUrl}${squadCategoriesPaths.discover}`;
```

Reply:
but the constant has an extra / in the beginning, which isn't a big deal but might cause anything unexpected

Reply:
Also, there is an extra `/` character if we use both together. Although this is for extension and the links here are only used on tablet. Shouldn't be too much of an issue I believe.

Reply:
We can maybe use some string replacement rather?
I just want to avoid having magic strings that are actually const elsewhere 🙈

Reply:
Yeah, I agree. Always better to use defined consts.